src/pages/registration/resource_type_tab/components/AnnouncementTableMoveButtons.tsx

```diff
@@ -3,21 +3,21 @@ import { Box, IconButton } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import { dataTestId } from '../../../../utils/dataTestIds';
 
-interface MusicalWorkMoveButtonsProps {
+interface AnnouncementTableMoveButtonsProps {
   index: number;
   listLength: number;
   moveItem: (to: number) => void;
 }
 
-export const MusicalWorkMoveButtons = ({ index, listLength, moveItem }: MusicalWorkMoveButtonsProps) => {
+export const AnnouncementTableMoveButtons = ({ index, listLength, moveItem }: AnnouncementTableMoveButtonsProps) => {
   const { t } = useTranslation();
 
   return (
     <Box sx={{ display: 'flex', alignSelf: 'center' }}>
```

Comment:
skulle vi hatt en maks-bredde e.l. for dette, for å ungå at innholdet man faktisk er interessert i (navn feks) kan havne langt til høyre i tabellen?

![Image](https://github.com/user-attachments/assets/1751e0ab-a893-46b6-b8d5-2096dab26657)